website/docs/introduction/community.md

```diff
@@ -6,80 +6,74 @@ sidebar_label: Community
 
 ---
 
+## Join the Litmus community
+
 Created by our users, contributors, and adopters, the Litmus community truly believes in adding to the value of open source culture. Serving and listening to the community is a priority to us.
 
-We welcome Kubernetes SREs and Developers to join this zealous community to learn and share all about Cloud-Native Chaos Engineering. There are many contributors in our community who are ready to mentor those who want to contribute. Ask a question in [our slack channel](https://slack.litmuschaos.io/).
+We invite Kubernetes SREs and Developers to join this community to learn and share knowledge about Cloud-Native Chaos Engineering. Many in our community are ready to mentor those who want to contribute. This page lists Litmus information and community resources.
+
+### Join our Slack channel
 
-## Join our Slack channel
+Join Our Community On Slack For detailed discussion, feedback, and regular updates on Chaos Engineering For Kubernetes. Join `#litmus` on `Kubernetes Slack` and connect with SREs and Chaos Engineering enthusiasts.
 
-Join Our Community On Slack For Detailed Discussion, Feedback & Regular Updates On Chaos Engineering For Kubernetes, join `#litmus` on `Kubernetes Slack` and connect with SREs and Chaos Engineering enthusiasts.
+Join [here](https://slack.litmuschaos.io/).
 
-Join here: https://slack.litmuschaos.io/
+### Attend our community meetings
```

Comment:
Could you add more content about community and contributor meetings based on this [information](https://github.com/litmuschaos/litmus?tab=readme-ov-file#community-meetings)?